js/anxpro.js

```diff
@@ -80,6 +122,327 @@ module.exports = class anxpro extends Exchange {
+        //     "transactionType": "DEPOSIT",
+        //     "received": "1529420056000",
+        //     "processed": "1529420766000",
+        //     "timestampMillis": "1557442743854",
+        //     "displayTitle": "Coin Deposit",
+        //     "displayDescription": "Deposit to: 0xf123aa44fadea913a7da99cc2ee202db684ce0e3",
+        //     "coinTransactionId": "0x33a3e5ea7c034dc5324a88aa313962df0a5d571ab4bcc3cb00b876b1bdfc54f7",
+        //     "coinConfirmations": 51,
+        //     "coinConfirmationsRequired": 45,
+        //     "subAccount": {"uuid": "aba1de05-c7c6-49d7-84ab-a6aca0e827b6", "name": "DEFAULT"}
+        //    }
+        const timestamp = this.safeInteger (transaction, 'received');
+        const updated = this.safeInteger (transaction, 'processed');
+        const transactionType = this.safeString (transaction, 'transactionType');
+        let type = undefined;
+        let amount = this.safeFloat (transaction, 'amount');
+        let address = this.safeString (transaction, 'coinAddress');
+        let tag = undefined;
+        if (transactionType === 'WITHDRAWAL') {
+            type = 'withdrawal';
+            amount = -amount;
+            if (address) {
+                //  xrp: "coinAddress": "rw2ciyaNshpHe7bCHo4bRWq6pqqynnWKQg?dt=3750180345",
+                if (address.indexOf ('?dt=') >= 0) {
+                    const parts = address.split ('?dt=');
+                    address = parts[0];
+                    tag = parts[1];
+                }
+            }
+        } else if (transactionType === 'DEPOSIT') {
+            if (!address) {
+                const displayDescription = this.safeString (transaction, 'displayDescription');
+                const addressText = displayDescription.replace ('Deposit to: ', '');
+                if (addressText.length > 0) {
+                    //  eth: "displayDescription": "Deposit to: 0xf123aa44fadea913a7da99cc2ee202db684ce0e3",
+                    //  xrp: "displayDescription": "Deposit to: rUjxty1WWLwX1evhKf3C2XNZDMcXEZ9ToJ?dt=504562345",
+                    if (addressText.indexOf ('?dt=') >= 0) {
+                        const parts = addressText.split ('?dt=');
+                        address = parts[0];
+                        tag = parts[1];
+                    } else {
+                        address = addressText;
+                    }
+                }
+            }
+            type = 'deposit';
+        }
+        const ccy = this.safeString (transaction, 'ccy');
+        let code = ccy;
+        currency = this.currency (ccy);
+        if (currency)
+            code = currency.id;
+        let transactionState = this.safeString (transaction, 'transactionState');
+        let status = 'pending';
+        if (transactionState === 'PROCESSED')
+            status = 'complete';
+        else if (transactionState === 'REVERSED' || transactionState === 'CANCELLED_INSUFFICIENT_FUNDS' || transactionState === 'CANCELLED_LIMIT_BREACH')
```

Comment:
I suggest to add parseTransactionStatus in here, see the [example here](https://github.com/ccxt/ccxt/blob/master/js/kraken.js#L1134) (can be copied literally, with very little changes)